standards/container_standards.md

```diff
@@ -0,0 +1,49 @@
+# Container standards
+
+## Terminology
+`Dockerfile` - set of instructions for building a docker image  
+`Image` - a constructed set of layered docker instructions  
+`Container` - a running instance of an image
+
+## Standards
+### Images are created using Docker
```

Comment:
Can we provide a one line rationale for this? Something like:
> Docker is the de facto industry standard in most enterprises.

Reply:
Added